api/app/wildfire_one/query_builders.py

```diff
@@ -66,8 +66,25 @@ def query(self, page) -> Tuple[str, dict]:
         return url, params
 
 
-class BuildQueryDailesByStationCode(BuildQuery):
-    """ Builds query for requesting all hourlies in a time range"""
+class BuildQueryAllForecastsByAfterStart(BuildQuery):
+    """ Builds query for requesting all dailies in a time range"""
+
+    def __init__(self, start_timestamp: int):
+        """ Initialize object """
+        super().__init__()
+        self.querystring: str = "weatherTimestamp >=" + \
```

Comment:
Can you switch this to using pythons new f-string pls?
```suggestion
        self.querystring = f"weatherTimestamp >= {start_timestamp};recordType.id == 'FORECAST'"
```